### ISSUE TITLE
Left and Top Nav-bar Issue

### ISSUE THREAD

**Issue by Mastermaulik** [openMF/community-app#1955]
This issue refers to new_reskin branch.

The left nav-bar seems to be short on some pages, here is an example:

<img width="1276" alt="screen shot 2017-03-01 at 8 01 05 am" src="https://cloud.githubusercontent.com/assets/5520612/23443520/42da079a-fe55-11e6-9715-7f020b797832.png">

The nav-bar should be fixed and should look like:
<img width="1280" alt="screen shot 2017-03-01 at 8 02 20 am" src="https://cloud.githubusercontent.com/assets/5520612/23443543/6dd01d54-fe55-11e6-8f4d-aa4a000902c3.png">

The nav-bar should be always fixed on the left side like in the second picture.

Also, the top nav bar disappears if we scroll till end:

Example:
<img width="1280" alt="screen shot 2017-03-01 at 8 24 59 am" src="https://cloud.githubusercontent.com/assets/5520612/23444035/9e36596a-fe58-11e6-811b-e125143bcd92.png">

versus:
<img width="1279" alt="screen shot 2017-03-01 at 8 25 49 am" src="https://cloud.githubusercontent.com/assets/5520612/23444058/b6191874-fe58-11e6-9311-3ac19134c019.png">

In short, both the nav-bars should be fixed and not move when the user scrolls

**Comment by skprabhanjan** [comment]
I will try working on this.

**Comment by Mastermaulik** [comment]
@skprabhanjan , this is a really high priority issue, so please try to get to the task ASAP.

**Comment by botraunak** [comment]
I'll try this issue!

**Comment by mbj36** [comment]
Fixed
